src/lib/components/profile/add/Add.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+  import addIcon from '$lib/icons/add-white.svg'
+
+  export let clickHandler: () => void
+</script>
+
+<button
+  on:click={clickHandler}
+  class="flex items-center justify-center gap-2 rounded-[10px] bg-ucab-green py-1 pl-5 pr-6 text-brand-white shadow-lg transition-all duration-200 hover:bg-green-600 hover:shadow-lg hover:shadow-emerald-200"
+>
+  <img src={addIcon} alt="Añadir enlace de interés" />
+  <p class="font-poppins text-[20px] font-medium">Añadir</p>
+</button>
```

Comment:
```suggestion
<script lang="ts">
  import addIcon from '$lib/icons/add-white.svg'

  export let clickHandler: () => void
  export let altImgText = 'Añadir'
</script>

<button
  on:click={clickHandler}
  class="flex items-center justify-center gap-2 rounded-[10px] bg-ucab-green py-1 pl-5 pr-6 text-brand-white shadow-lg transition-all duration-200 hover:bg-green-600 hover:shadow-lg hover:shadow-emerald-200"
>
  <img src={addIcon} alt={altImgText} />
  <p class="font-poppins text-[20px] font-medium">Añadir</p>
</button>
```
Aplica esta corrección si este botón de añadir es uno genérico dentro del perfil y puede estar en cualquier sección, ya que el texto alternativo a la imagen va a depender de dónde esté entonces

Reply:
arreglo